crest/Assets/Crest/Crest/Scripts/Helpers/UnderwaterCamera.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Crest
+{
+
+    [RequireComponent(typeof(Camera))]
+    public class UnderwaterCamera : MonoBehaviour
+    {
```

Comment:
We should probably add a longform comment here explaining how everything is meant to work.

Reply:
Had a stab, feel free to review